lib/http.c

```diff
@@ -3054,7 +3054,7 @@ CURLcode Curl_http_readwrite_headers(struct SessionHandle *data,
         switch(k->httpcode) {
         case 100:
           /* if we did wait for this do enable write now! */
-          if(k->exp100) {
+          if(k->exp100 > EXP100_SEND_DATA) {
```

Comment:
It's just a style change... now it's the same as in transfer.c line 449

Of course you may skip this change if you want.